doc/man3/SSL_shutdown.pod

```diff
@@ -104,11 +122,11 @@ and the peer's "close notify" alert was received.
 
 =item E<lt>0
 
-The shutdown was not successful because a fatal error occurred either
-at the protocol level or a connection failure occurred. It can also occur if
-action is need to continue the operation for non-blocking BIOs.
-Call L<SSL_get_error(3)> with the return value B<ret>
-to find out the reason.
+The shutdown was not successful.
+Call L<SSL_get_error(3)> with the return value B<ret> to find out the reason.
+It can occur if action is needed to continue the operation for non-blocking BIOs.
```

Comment:
not sure if the sentence would sound better this way:
"It can occur if an action is needed..." ?